.travis.yml

```diff
@@ -20,12 +20,11 @@ before_install:
   - sudo apt-get install software-properties-common
   - sudo add-apt-repository -y ppa:ethereum/ethereum
   - sudo apt-get update
-  - sudo apt-get --allow-unauthenticated install solc
 install:
   - npm install
 before_script:
   - nohup sh tools/runGanacheCli.sh </dev/null >/dev/null 2>&1 &
-  - bash tools/compile.sh
+  - truffle compile
```

Comment:
Travis will fail if we remove **truffle compile** because it will use solidity compiler version 5.
https://travis-ci.org/OpenSTFoundation/brandedtoken-contracts/builds/464819486?utm_source=github_status&utm_medium=notification